### PR TITLE
Adding support to check if an attribute is present or not without having to get the value

### PR DESCRIPTION
**Pull request by shoubhik**
There is a requirement in the parsing of pre-quantized mxnet model where; an attribute's value depends on another attribute being present or absent. The example can be found here
1. https://github.com/apache/incubator-mxnet/blob/master/src/operator/quantization/quantize_v2.cc#L87-L90
2. https://t.ly/AKMLm

Since there is no way for querying if an attribute is present or not in a very clean way possible I am adding a new method to check if an attribute is present.

Since the attribute in question is a float passing in a NAN or something equivalent is not right since i NAN could be a legal value of something gone wrong during quantization.
Another way could be to pass a String instead as a default value, but that will not be a clean solution since we have a api which is supposed to return a float is returning a string instead.

### CODE REVIEW

**Comment by shoubhik** [comment]
@zhiics @vinx13 @anijain2305 Can you please review?

File: python/tvm/relay/frontend/common.py [hunk 1]
```diff
@@ -41,6 +41,20 @@ class StrAttrsDict(object):
     def __init__(self, attrs):
         self.attrs = attrs
 
+    def is_attr_present(self, key):
```
Comment (vinx13) [inline python/tvm/relay/frontend/common.py]: let's think about the name, shall we just name it `has_attr`?

Comment (shoubhik) [reply python/tvm/relay/frontend/common.py]: I don't have a problem with `has_attr`. i can submit a change with that name.

**Comment by shoubhik** [comment]
@vinx13 can you take a look at it now.

**Pull request merged**